src/map/handler/Map.Gesture.js

```diff
@@ -0,0 +1,102 @@
+import {Map} from '../Map';
+import {Handler} from '../../core/Handler';
+import * as DomEvent from '../../dom/DomEvent';
+import * as Util from '../../core/Util';
+import * as DomUtil from '../../dom/DomUtil';
+
+/*
+ * L.Handler.Gesture is used by L.Map to add gesture based pan and zoom on toupad based devices.
```

Comment:
As I see you've renamed handler from `gesture` to `touchpad`.
That means that you should rename source file too, and all related identifiers.

And I still don't feel that name `touchpad` is specific enough.